File: frontend/src/features/actividades/verificar_reparo_estudio_titulo/models/verificar_reparo_estudio_titulo.ts
export interface VerificarReparoEstudioTitulo {
  id_verificar_reparo_estudio_titulo: number;
  id_expediente: number;
  enviar_a_reparo: boolean;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/verificar_reparo_estudio_titulo.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class verificar_reparo_estudio_titulo : base_auditoria
    {
        public int id_verificar_reparo_estudio_titulo { get; set; }
        public long id_expediente { get; set; }
        public bool? enviar_a_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}
